src/Sylius/Behat/Context/Setup/UserContext.php

```diff
@@ -65,19 +76,108 @@ public function __construct(
      */
     public function thereIsUserIdentifiedBy($email, $password)
     {
-        /** @var UserInterface $user */
-        $user = $this->userFactory->createNew();
+        $customer = $this->createCustomer();
+        $user = $this->createUser($customer, $email, $password);
+
+        $this->sharedStorage->setCurrentResource('user', $user);
+        $this->userRepository->add($user);
+    }
+
+    /**
+     * @Given there is user :email identified by :password, with :country as shipping country
+     */
+    public function thereIsUserWithShippingCountry($email, $password, $country)
+    {
+        $customer = $this->createCustomer();
+        $user = $this->createUser($customer, $email, $password);
+
+        $customer->setShippingAddress($this->createAddress($customer->getFirstName(), $customer->getLastName(), $country));
+
+        $this->sharedStorage->setCurrentResource('user', $user);
+        $this->userRepository->add($user);
+    }
+
+    /**
+     * @param string $firstName
+     * @param string $lastName
+     *
+     * @return CustomerInterface
+     */
+    private function createCustomer($firstName = 'John', $lastName = 'Doe')
+    {
         $customer = $this->customerFactory->createNew();
-        $customer->setFirstName('John');
-        $customer->setLastName('Doe');
+        $customer->setFirstName($firstName);
+        $customer->setLastName($lastName);
+
+        return $customer;
+    }
+
+    /**
+     * @param CustomerInterface $customer
+     * @param string $email
+     * @param string $password
+     *
+     * @return UserInterface
+     */
+    private function createUser(CustomerInterface $customer, $email = 'john.doe@example.com', $password = 'testPassword')
```

Comment:
Please be consistent ;) Once you use multi-line (see below) once you put it all in one line :)